src/js/components/stock-movement-wizard/inbound/AddItemsPage.jsx

```diff
@@ -597,7 +600,7 @@ class AddItemsPage extends Component {
   saveAndTransitionToNextStep(formValues, lineItems) {
     this.props.showSpinner();
 
-    this.saveRequisitionItems(lineItems)
+    this.saveRequisitionItemsInCurrentStep(lineItems)
```

Comment:
Why did we need to change this method?

Reply:
Never mind, I see there are two methods. I would prefer keeping the name saveRequisitionItems rather than saveRequisitionItemsInCurrentStep().